nservicebus/sqlserver/deployment-options.md

```diff
@@ -0,0 +1,58 @@
+---
+title: SQL Server transport deployment options
+summary: Describes available SQL Server transport deployment options
+tags:
+- SQL Server
+- Transports
+redirects:
+- nservicebus/sqlserver/multiple-databases
+---
+
+In default configuration SQL Server transport assumes that all tables used for storing messages are located in a single catalog and within a single schema. The configuration can be changed to partition message storage between different schemas and catalogs. The decision can me made on per queue basis (including error and audit queues).
+
+The supported deployment options are:
+ * ***default***: all queues are stored in a single catalog and a single schema (Versions 1.x and higher)
+ * ***multi-schema***: queues are stored in a single catalog but in more than one schema (Versions 2.1 and higher)
+ * ***multi-catalog***: queues are stored in multiple catalogs but on a single SQL Server instance. This mode is not explicitly supported by SQL Server transport, but can be achieved  by using multi-instance option. Therefore in this document both options are covered under *multi-instance* term.
+ * ***multi-instance***: queues are stored in multiple catalogs on more than one SQL Server instance (Versions 2.1 to 2.x).
```

Comment:
Wasn't multi instance already there in an earlier version? I suspect it is supported since 1.2